administrator/components/com_workflow/src/Controller/StagesController.php

```diff
@@ -195,4 +195,51 @@ protected function getRedirectToListAppend()
     {
         return '&extension=' . $this->extension . ($this->section ? '.' . $this->section : '') . '&workflow_id=' . $this->workflowId;
     }
+
+    /**
+     * Update positions for multiple workflow stages
+     *
+     * @return  void
+     *
+     * @since   4.0.0
+     */
+    public function updatePositions()
```

Comment:
Isn't this a theoretical duplicate of the save method in the Graph controller and can be merged?